test/over_react/util/ddc_emulated_function_name_bug_test.dart

```diff
@@ -21,7 +21,7 @@ main() {
     group('isBugPresent correctly detects whether the bug is present', () {
       test('in dart2js', () {
         expect(isBugPresent, isFalse);
-      }, testOn: 'js', tags: 'no-ddc');
+      }, testOn: 'js');
```

Comment:
We actually should be able to rip out this library since https://github.com/dart-lang/sdk/issues/29904 is fixed in Dart 2!

Can we do that (probably in a separate PR)?

Reply:
Yes, I'll cut a ticket

Reply:
Forgot to link ticket: https://jira.atl.workiva.net/browse/AF-3974